search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt

```diff
@@ -100,6 +100,10 @@ class TemporalEntityHandler(
             temporalEntitiesQuery.queryParams.attrs.isEmpty()
         ) throw BadRequestDataException("Either type or attrs need to be present in request parameters")
 
+        if (temporalEntitiesQuery.temporalQuery.timeAt == null &&
+            temporalEntitiesQuery.temporalQuery.timerel == null
+        ) throw BadRequestDataException("Either timerel and timeAt need to be present in request parameters")
+
```

Comment:
it would be better that the temporal query be fully validated in `QueryUtils.buildTemporalQuery()` (as it is currently the case, you are spreading its validation in two places)